yetanotherco/aligned_layer
# feat(explorer): cost per proof chart

## Description
- Integrate [chart.js](https://www.chartjs.org/) library and create `line_chart` component.
- Fetch `fee_per_proof` for the last `n` `batches`
- Add a new module in `home` that shows the cost per proof of the last `100` batches

<img width="1116" alt="Screenshot 2024-12-19 at 2 48 35 PM" src="https://github.com/user-attachments/assets/2544b755-60c0-4e24-a83c-b3f4ced26ef8" />

**Testing**
- Start a local devnet.
- Run the explorer.
- Send proofs and see how the graph starts to fill. 

## Type of change



## Checklist

  - [Link to the open issue addressing it]()
